docs/source/variational-problems.rst

```diff
@@ -499,8 +499,8 @@ for the various :ref:`utility meshes <utility_mesh_functions>` are
 described in their respective constructor documentation.  For
 externally generated meshes, Firedrake just uses whichever ids the
 mesh generator provided.  The ``value`` may be either a scalar, or
-more generally an :py:class:`~.Expression`, :py:class:`~.Function` or
-:py:class:`~.Constant` of the appropriate shape.  You may also supply
+more generally a UFL expression, for example and :class:`~.Function` or
+:py:class:`~.Constant`, of the appropriate shape.  You may also supply
```

Comment:
Everywhere you've changed the docs to only say "UFL expression" you might as well say 
```
for example and :class:`~.Function` or :py:class:`~.Constant`
```
for completeness.
We don't want people to look at the API documentation for something like interpolation and think they can only interpolate pure UFL.